yo1k/tic_tac_toe/core/game.py

```diff
@@ -256,23 +259,23 @@ def __last_step(step: int) -> bool:
         return step == Board.size() ** 2 - 1
 
     @staticmethod
-    def __surrender(state: State):
+    def __surrender(state: State) -> None:
         assert state.phase is Phase.INROUND
         idx_other_player = (state.turn() + 1) % 2
```

Comment:
Use `% len(state.players)` instead of 2.

Reply:
Done.